### PR TITLE
fix(console): password validation hints, mfa, i18n

### PR DESCRIPTION
**Pull request by peintnermax**

### CODE REVIEW

File: console/src/assets/i18n/de.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
             "DETAIL": "Detail",
             "CREATE": "Erstellen",
             "MY": "Meine Informationen",
-            "LOGINNAMES":"Loginnamen",
+            "LOGINNAMES":"Zugangsmethoden",
```
Comment (fforootd) [inline console/src/assets/i18n/de.json:55]:
```suggestion
            "LOGINNAMES":"Benutzernamen",
```

@fgerschwiler do you have a better idea?

Comment (fforootd) [reply console/src/assets/i18n/de.json:55]: or is it `Loginname`?

File: console/src/assets/i18n/en.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
             "DETAIL": "Detail",
             "CREATE": "Create",
             "MY": "My Informations",
-            "LOGINNAMES":"Login names",
+            "LOGINNAMES":"Methods to access your profile",
```
Comment (fforootd) [inline console/src/assets/i18n/en.json:55]: Does this make sense?
I don't understand this one 😁

File: console/src/assets/i18n/de.json (hunk 1 above)
Comment (peintnermax) [reply console/src/assets/i18n/de.json:55]: @fgerschwiler would name it 'Loginnamen'

**Comment by fforootd** [comment]
Make it so ;-)

Florian Forster

H e a d   o f C A O S

Phone:  +41 79 956 39 01

Web:      www.caos.ch

File: console/src/assets/i18n/en.json (hunk 2 above)
Comment (peintnermax) [reply console/src/assets/i18n/en.json:55]: its a the card header title under userinfo to display methods to access you profile.
We should keep it login names as long as there are no other methods like an authenticator app 👍

**Pull request merged**